prow/test/data/test-config.yaml

```diff
@@ -1,7 +1,7 @@
 plank:
   default_decoration_config:
     timeout: 7200000000000 # 2h
```

Comment:
You want to change this?

Reply:
No, I want to test that the old format parses correctly too. I validate that the `timeout` field is parsed correctly in the explicit value test case.